### PR TITLE
Fix casa compile bug by pinning the dependencies

### PR DESCRIPTION
**Pull request by gevos**
# How to

See TRAFCASA-73

# Others

- [x] It's not a hack or at least an unauthorized hack :).
- [x] Everything in ticket description has been fixed.
- [x] The author of the MR has made its own review before assigning the reviewer.
- [ ] IE11 tested.
- [x] The title means something for a human being.
- [ ] Labels applied. if it's a release? a hotfix?
- [ ] Tests added.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -113,7 +113,7 @@
     "sort": "sort-json src/lang/*.json -i",
     "start": "react-app-rewired start",
     "start:doc": "styleguidist server",
-    "test": "react-app-rewired test  --env=jsdom --transformIgnorePatterns \"/node_modules/(?!(@geops/react-ui|jsts|ol|react-spatial|mobility-toolbox-js)/).*/\""
+    "test": "react-app-rewired test  --env=jsdom --transformIgnorePatterns \"/node_modules/(?!(@geops/react-ui|jsts|ol|react-spatial|mobility-toolbox-js|ol-mapbox-style)/).*/\""
```
Comment (gevos) [inline package.json:116]: I had to add `ol-mapbox-style` to the `transformIgnorePatterns`, see commit message.

**Review by oterral — CHANGES_REQUESTED** [review]
Pleave leave versions  pinned but change just the version number. With unpinned version we had cobnflict version trouble in the past.

**Comment by gevos** [comment]
> Pleave leave versions pinned but change just the version number. With unpinned version we had conflict version trouble in the past.

Oh, shit. Fixed.

Test with https://github.com/oterral/angular-react-test was successful.

File: yarn.lock [hunk 2]
```diff
@@ -1373,15 +1373,19 @@
     react-shadow "^17.1.3"
     rimraf "^3.0.0"
 
-"@geops/geops-ui@0.1.8":
-  version "0.1.8"
-  resolved "https://registry.yarnpkg.com/@geops/geops-ui/-/geops-ui-0.1.8.tgz#4df8f4bb8916d762e8b9667cf744fa8961c3598d"
-  integrity sha512-ms3zDcXThMEoVP4wswdMZNajIlwr1mBlFDmtoe4qS1eN7b6iBgd3I2jJ1zSRSbuhaKMmyw7KX+iJcLv3/jY0zg==
+"@geops/geops-ui@0.1.4":
+  version "0.1.4"
```
Comment (oterral) [inline yarn.lock:1377]: You have to pin to the last version we used . otherwise we regress.

File: yarn.lock [hunk 3]
```diff
@@ -2446,16 +2455,16 @@ abbrev@1:
   resolved "https://registry.yarnpkg.com/abbrev/-/abbrev-1.1.1.tgz#f8f2c887ad10bf67f634f005b6987fed3179aac8"
   integrity sha512-nne9/IiQ/hzIhY6pdDnbBtz7DjPTKrY00P/zvPSm5pOFkl6xuGrGnXn/VtTNNfNtAfZ9/1RtehkszU9qcTii0Q==
 
+abortcontroller-polyfill@1.4.0:
+  version "1.4.0"
```
Comment (oterral) [inline yarn.lock:2459]: regression

File: yarn.lock [hunk 4]
```diff
@@ -2446,16 +2455,16 @@ abbrev@1:
...
+  resolved "https://registry.yarnpkg.com/abortcontroller-polyfill/-/abortcontroller-polyfill-1.4.0.tgz#0d5eb58e522a461774af8086414f68e1dda7a6c4"
+  integrity sha512-3ZFfCRfDzx3GFjO6RAkYx81lPGpUS20ISxux9gLxuKnqafNcFQo59+IoZqpO2WvQlyc287B62HDnDdNYRmlvWA==
+
 abortcontroller-polyfill@1.5.0:
   version "1.5.0"
```
Comment (oterral) [inline yarn.lock:2464]: Why there is 2 abortcontroller-polyfill?  that's wierd

File: yarn.lock [hunk 5]
```diff
@@ -8473,24 +8482,24 @@ hyphenate-style-name@^1.0.2, hyphenate-style-name@^1.0.3:
...
   integrity sha512-OtRf2Vo3IqAxsttQbpjYnmMML12IMB5e0fc5B7qKJFLScitYaXa1OhMX0n0X/3vrfFlpHL9Ro/H+ps4Ej2j7QQ==
   dependencies:
     "@babel/runtime" "^7.5.5"
 
-i18next@^19.3.3:
-  version "19.3.4"
-  resolved "https://registry.yarnpkg.com/i18next/-/i18next-19.3.4.tgz#512de50ee6075df825c646e1ce646a104f0938c9"
-  integrity sha512-ef7AxxutzdhBsBNugE9jgqsbwesG1muJOtZ9ZrPARPs/jXegViTp4+8JCeMp8BAyTIo1Zn0giqc8+2UpqFjU0w==
+i18next@19.3.3:
+  version "19.3.3"
```
Comment (oterral) [inline yarn.lock:8500]: regression

File: yarn.lock [hunk 6]
```diff
@@ -10675,6 +10684,11 @@ lodash.uniq@^4.5.0:
   resolved "https://registry.yarnpkg.com/lodash.uniq/-/lodash.uniq-4.5.0.tgz#d0225373aeb652adc1bc82e4945339a842754773"
   integrity sha1-0CJTc662Uq3BvILklFM5qEJ1R3M=
 
+lodash@4.17.11:
+  version "4.17.11"
```
Comment (oterral) [inline yarn.lock:10688]: regression

File: yarn.lock [hunk 7]
```diff
@@ -13625,13 +13662,13 @@ progress@^2.0.0:
   resolved "https://registry.yarnpkg.com/progress/-/progress-2.0.3.tgz#7e8cf8d8f5b8f239c1bc68beb4eb78567d572ef8"
   integrity sha512-7PiHtLll5LdnKIMw100I+8xJXR5gW2QwWYkT6iJva0bXitZKa/XMrSbdmg3r2Xnaidz9Qumd0VPaMrZlF9V9sA==
 
-proj4@^2.6.0:
-  version "2.6.1"
-  resolved "https://registry.yarnpkg.com/proj4/-/proj4-2.6.1.tgz#c07a32cd4a38a82515aae3b54363669896641dd6"
-  integrity sha512-RP5EcrfrLcARy+Zjjz1wIeqZzZdPtQNl685asHcwdU/MQ/dvydmf1XWM4mgok6wPaNsXZ8IFrM4qadO3g46PiQ==
+proj4@2.6.0:
+  version "2.6.0"
```
Comment (oterral) [inline yarn.lock:13666]: regression

File: yarn.lock [hunk 8]
```diff
@@ -13670,6 +13707,14 @@ prop-types-exact@^1.2.0:
     object.assign "^4.1.0"
     reflect.ownkeys "^0.2.0"
 
+prop-types@15.6.2:
+  version "15.6.2"
```
Comment (oterral) [inline yarn.lock:13711]: regression

File: yarn.lock [hunk 9]
```diff
@@ -13839,6 +13884,15 @@ qss@^2.0.3:
   resolved "https://registry.yarnpkg.com/qss/-/qss-2.0.3.tgz#630b38b120931b52d04704f3abfb0f861604a9ec"
   integrity sha512-j48ZBT5IZbSqJiSU8EX4XrN8nXiflHvmMvv2XpFc31gh7n6EpSs75bNr6+oj3FOLWyT8m09pTmqLNl34L7/uPQ==
 
+query-string@6.11.1:
+  version "6.11.1"
```
Comment (oterral) [inline yarn.lock:13888]: regression

File: yarn.lock [hunk 10]
```diff
@@ -14376,14 +14430,13 @@ react-simple-code-editor@^0.10.0:
   resolved "https://registry.yarnpkg.com/react-simple-code-editor/-/react-simple-code-editor-0.10.0.tgz#73e7ac550a928069715482aeb33ccba36efe2373"
   integrity sha512-bL5W5mAxSW6+cLwqqVWY47Silqgy2DKDTR4hDBrLrUqC5BXc29YVx17l2IZk5v36VcDEq1Bszu2oHm1qBwKqBA==
 
-react-spatial@1.0.11:
-  version "1.0.11"
-  resolved "https://registry.yarnpkg.com/react-spatial/-/react-spatial-1.0.11.tgz#7ddbdb0fc1964b47894d72373bd897cc024d0f21"
-  integrity sha512-m4P4ZM/m5M10rmPYj8eZMEH2CnceR6uoCGem3jqeHU0N4SREMi1vU6KdinvK9Ev3O374vWTszqc0zJOI5UHA9A==
+react-spatial@1.0.8:
+  version "1.0.8"
```
Comment (oterral) [inline yarn.lock:14434]: regression

File: yarn.lock [hunk 11]
```diff
@@ -14394,10 +14447,10 @@ react-spatial@1.0.11:
     react-icons "3.10.0"
     resize-observer-polyfill "1.5.1"
 
-react-styleguidist@^11.0.4:
-  version "11.0.5"
-  resolved "https://registry.yarnpkg.com/react-styleguidist/-/react-styleguidist-11.0.5.tgz#b1a1eb91f8bf7c538fc0d652f83c2d8cd997f5aa"
-  integrity sha512-LsLZ4aIN1gsBRgRJcEmOAoobdwVMJbnhG+49F4CgKDuuYkPazkJ1l4GuBcMcOSNfiLz/vA2nkl3TpJHH/7CVug==
+react-styleguidist@11.0.4:
+  version "11.0.4"
```
Comment (oterral) [inline yarn.lock:14451]: regression

File: yarn.lock [hunk 12]
```diff
@@ -16645,14 +16708,14 @@ style-search@^0.1.0:
   resolved "https://registry.yarnpkg.com/style-search/-/style-search-0.1.0.tgz#7958c793e47e32e07d2b5cafe5c0bf8e12e77902"
   integrity sha1-eVjHk+R+MuB9K1yv5cC/jhLneQI=
 
-styled-components@^5.0.1:
-  version "5.1.0"
-  resolved "https://registry.yarnpkg.com/styled-components/-/styled-components-5.1.0.tgz#2e3985b54f461027e1c91af3229e1c2530872a4e"
-  integrity sha512-0Qs2wEkFBXHFlysz6CV831VG6HedcrFUwChjnWylNivsx14MtmqQsohi21rMHZxzuTba063dEyoe/SR6VGJI7Q==
+styled-components@5.0.1:
+  version "5.0.1"
```
Comment (oterral) [inline yarn.lock:16712]: regression

File: yarn.lock (hunk 2 above)
Comment (gevos) [reply yarn.lock]: I think, I have all regression cases fixed

**Pull request merged**